junos/resource_security_nat_source_pool.go

```diff
@@ -45,6 +47,16 @@ func resourceSecurityNatSourcePool() *schema.Resource {
 				MinItems: 1,
 				Elem:     &schema.Schema{Type: schema.TypeString},
 			},
+			"address_pooling_no_paired": {
+				Type:          schema.TypeBool,
+				Optional:      true,
+				ConflictsWith: []string{"address_pooling_paired"},
+			},
+			"address_pooling_paired": {
+				Type:          schema.TypeBool,
+				Optional:      true,
+				ConflictsWith: []string{"address_pooling_no_paired"},
+			},
```

Comment:
Could you instead use a single `address_pooling` argument with 2 choices `paired` and `no-paired`  ?
```suggestion
			"address_pooling": {
				Type:         schema.TypeString,
				Optional:     true,
				ValidateFunc: validation.StringInSlice([]string{"no-paired", "paired"}, false),
			},
```